scene/2d/cpu_particles_2d.cpp

```diff
@@ -1211,6 +1215,27 @@ void CPUParticles2D::_notification(int p_what) {
 			}
 
 			RS::get_singleton()->canvas_item_add_multimesh(get_canvas_item(), multimesh, texrid);
+
+#ifdef TOOLS_ENABLED
```

Comment:
You also need to use `is_editor_hint()`, otherwise the gizmo draws at runtime...